extensions/vertx-web/runtime/src/main/java/io/quarkus/vertx/web/runtime/cors/CORSConfig.java

```diff
@@ -37,19 +42,32 @@
      * default: returns any requested header as valid
      */
     @ConfigItem
-    public Optional<String> headers;
+    public List<String> headers;
+
     /**
      * HTTP headers exposed in CORS
      *
      * Comma separated list of valid headers. ex: X-Custom,Content-Disposition
-     * 
+     *
      * default: <empty>
      */
     @ConfigItem
-    public Optional<String> exposedHeaders;
+    public List<String> exposedHeaders;
+
+    /**
+     * The `Access-Control-Max-Age` response header value indicating
+     * how long the results of a pre-flight request can be cached.
+     */
+    @ConfigItem
+    public Optional<Duration> accessControlMaxAge;
```

Comment:
You don't want to include it in the `toString()`?

Reply:
Well spotted. I have added it.